docs/model-types.mdx

```diff
@@ -0,0 +1,467 @@
+A good example is the [Customer Churn model](/sql/tutorials/customer-churn#training-a-predictor/).
+
+- [If it is a time series case](https://github.com/mindsdb/lightwood/blob/9a48d5523d78111c77024e1983d281f37a9782ea/lightwood/api/json_ai.py#L265,L277), we use the [NeuralTs](https://lightwood.io/mixer.html#mixer.NeuralTs) mixer.
+A good example is the [House Sales model](/sql/tutorials/house-sales-forecasting#training-a-predictor/).
+
+</Tab>
+
+<Tab title="Case 2">
+
+Here is the [piece of code](https://github.com/mindsdb/lightwood/blob/9a48d5523d78111c77024e1983d281f37a9782ea/lightwood/api/json_ai.py#L279,L324) being analyzed.
+
+[If it is not a time series case, or it is a time series case with the HORIZON value being one, and the target variable type is not an array of values](https://github.com/mindsdb/lightwood/blob/9a48d5523d78111c77024e1983d281f37a9782ea/lightwood/api/json_ai.py#L279,L310), we use the [LightGBM](https://lightwood.io/mixer.html#mixer.LightGBM), [XGBoostMixer](https://lightwood.io/mixer.html#mixer.XGBoostMixer), [Regression](https://lightwood.io/mixer.html#mixer.Regression), and [RandomForest](https://lightwood.io/mixer.html#mixer.RandomForest) mixers.
+
+A good example is the [Home Rentals model](/sql/tutorials/home-rentals#training-a-predictor/).
+
+[Otherwise, if it is a time series case and the HORIZON value is greater than one](https://github.com/mindsdb/lightwood/blob/9a48d5523d78111c77024e1983d281f37a9782ea/lightwood/api/json_ai.py#L311,L324), we use the [LightGBMArray](https://lightwood.io/mixer.html#mixer.LightGBMArray) mixer.
+
+A good example is the [House Sales model](/sql/tutorials/house-sales-forecasting#training-a-predictor/).
+
+</Tab>
+
+<Tab title="Case 3">
+
+Here is the [piece of code](https://github.com/mindsdb/lightwood/blob/9a48d5523d78111c77024e1983d281f37a9782ea/lightwood/api/json_ai.py#L326,L351) being analyzed.
+
+[If it is an autoregressive case and the target type is an integer, float, or quantity](https://github.com/mindsdb/lightwood/blob/9a48d5523d78111c77024e1983d281f37a9782ea/lightwood/api/json_ai.py#L326,L351), we use the [SkTime](https://lightwood.io/mixer.html#mixer.SkTime), [ETSMixer](https://lightwood.io/mixer.html#mixer.ETSMixer), and [ARIMAMixer](https://lightwood.io/mixer.html#mixer.ARIMAMixer) mixers.
+
+The autoregressive case means that we use the target values (as encoded features) from as many previous rows as defined in the `WINDOW` clause.
+
+A good example is the [Home Rentals model](/sql/tutorials/home-rentals#training-a-predictor/).
+
+</Tab>
+
+</Tabs>
+
+<br></br>
+
+<Note>
+MindsDB may use one or multiple mixers while preparing a model. Depending on the model type and the data type of the target variable, one mixer or a set of mixers is chosen to create, train, and validate an ML model.
```

Comment:
We could mention ensembling here, fits well:

> [...] one mixer is chosen or a set of mixers are ensembled to create [...]

Reply:
Changed:

> <Note>
> MindsDB may use one or multiple mixers while preparing a model. Depending on the model type and the data type of the target variable, one mixer is chosen or a set of mixers are ensembled to create, train, and validate an ML model.
> </Note>